src/inverseRobot.js

```diff
@@ -4,10 +4,20 @@
  * @param {object} robot
  *
  * @return {object}
+ * 1.Object.entries transform object to array
+ * 2.map >= reversed key and value and it's still array
+ * 3.Object.fromEntries transform array back to object
 */
 
 function inverseRobot(robot) {
-  // write code here
+  const inversed
+    = Object.fromEntries(Object.entries(robot).map(([key, value]) =>
+      [value, key]));
+
+  return Object.keys(inversed).length
+    < Object.values(robot).length
+    ? null
+    : inversed;
```

Comment:
To mush operations.
Create `newRobot `as empty object.
get array (key, value) using `Object.entries`
use` for of` loop for array. 
use `newRobot[value] = key;` to add properties
add check: if newRobot already have this key, if so return null